web/app/js/components/TabbedMetricsTable.jsx

```diff
@@ -11,10 +11,10 @@ import { Table } from 'antd';
 const resourceInfo = {
   "upstream_deployment": { title: "deployment", url: "/deployment?deploy=" },
   "downstream_deployment": { title: "deployment", url: "/deployment?deploy=" },
-  "upstream_pod": { title: "upstream pod", url: "/pod?pod=" },
-  "downstream_pod": { title: "downstream pod", url: "/pod?pod=" },
+  "upstream_pod": { title: "upstream pod", url: null },
```

Comment:
should these lines be removed entirely?

Reply:
@adleong removing those lines entirely breaks the deployment detail page --- I wasn't sure if we still wanted to render the list of pods in the deployment. If we don't, I can remove that as well.

Reply:
you should be able to remove these lines

Reply:
whoops, totally forgot about that --- removed in a76ea00cc7b475dd04e22fa39cb692bef88b7949